### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by syedsimanta03** [reactjs/react.dev#2917]
Please check: https://reactjs.org/community/starter-kits.html

**Comment by tombrowndev** [comment]
@syedsimanta03 The tools section was removed in this here https://github.com/reactjs/reactjs.org/pull/2535

**Comment by rickhanlonii** [comment]
Yup, this was removed. It was barely used or maintained so there's nothing to redirect to.
